lib/SPIRV/OCLUtil.cpp

```diff
@@ -761,6 +761,22 @@ class OCLBuiltinFuncMangleInfo : public SPIRV::BuiltinFuncMangleInfo {
     } else if (UnmangledName.find("intel_sub_group_media_block_write") !=
                std::string::npos) {
       addUnsignedArg(3);
+    } else if (UnmangledName.find(kOCLBuiltinName::SubGroupPrefix) !=
```

Comment:
This is not for this PR, but I wonder why do we use `std::string` here? We could use `llvm::StringRef::contains` and significantly shorten all these ifs.

Reply:
I've had that cleanup on my TODO list for a long time indeed, but never got around to do it.  But that's something beyond the scope of this PR anyway.

Reply:
Ok, let's leave it as a future refactoring opportunity.